en/sim_sih/index.md

```diff
@@ -93,7 +93,7 @@ To display the simulated vehicle:
      On macOS this would be `/dev/tty.usbmodem1`.
    - `-b` to set the serial baud rate to `2000000`.
    - `-o` to start jMAVSim in _display Only_ mode (i.e. the physical engine is turned off and jMAVSim only displays the trajectory given by the SIH in real-time).
-   - add a flag `-a` to display an aircraft or '-t' to display a tailsitter.
+   - add a flag `-a` to display an aircraft or ’-t’ to display a tailsitter.
```

Comment:
This would be done using backticks like in the `a`, as that makes it code style.